plugins/docsettingtweak.koplugin/main.lua

```diff
@@ -0,0 +1,30 @@
+local WidgetContainer = require("ui/widget/container/widgetcontainer")
+local _ = require("gettext")
+--local logger = require("logger")
+local dirname = require("ffi/util").dirname
+local util = require("util")
+
+local DocSettingTweak = WidgetContainer:new{
+    name = 'docsettingtweak',
+}
+
+local directory_defaults = {
+}
+
+function DocSettingTweak:onDocSettingsLoad(doc_settings, docfile)
+    if next(doc_settings.data) == nil then
+        local base = G_reader_settings:readSetting("home_dir") or "/"
```

Comment:
instead of `or "/"`:
`or require("apps/filemanager/filemanagerutil").getDefaultDir()`